q2_sample_classifier/classify.py

```diff
@@ -101,6 +101,19 @@ def regress_samples(output_dir: str, table: pd.DataFrame,
                optimize_feature_selection, title='regression predictions')
 
 
+def split_table(table: pd.DataFrame, metadata: qiime2.MetadataColumn,
+                test_size: float=defaults['test_size'], random_state: int=None,
+                stratify: str=True) -> (pd.DataFrame, pd.DataFrame):
+    column = metadata.to_series().name
```

Comment:
Metadata API PSA: [`metadata.name`](https://dev.qiime2.org/latest/api-reference/metadata/#qiime2.metadata.MetadataColumn.name) will get you what you are looking for.

Reply:
👍 Thanks @thermokarst ! Fixed throughout.